prettyprinter/prettyprinter.cabal

```diff
@@ -82,12 +95,29 @@ library
     if !impl(ghc >= 7.10)
         build-depends: void >=0.4 && <0.8
 
-
+-- A fake text package, emulating the same API, but backed by String
+library fake-text
```

Comment:
I'm surprised that this internal library doesn't produce a warning in combination with `cabal-version: >= 1.10`. Has Cabal supported these for so long?! And `stack` can handle them too?!

(I'm just asking – I've never knowingly used an internal library before.)

Reply:
Ah, `cabal build` was silent, but `cabal check` actually mentions that this requires `cabal-version: 2.0`. Do you think it is a problem? I can revert running tests in `-f-text` mode.

AFAICT `stack` can handle it well, I used this feature in other packages for some time.

Reply:
I think we'll probably be fine with requiring `cabal-version: 2.0`. If it does turn out to cause trouble, we can still restore compatibility with `Cabal-1.10`.